hackathon_site/dashboard/frontend/package.json

```diff
@@ -37,8 +37,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "i": "yarn install --frozen-lockfile",
-    "pretty": "yarn prettier --write 'src/**/*.(scss|js)'",
-    "precheck": "yarn pretty && yarn test -o -u --watchAll=false"
+    "pretty": "yarn prettier --write 'src/**/*.(scss|js|ts)'",
```

Comment:
Gotta include `tsx` in here too. React typescript files have to be .tsx, can't use .ts like you can with .js